cli/src/api/mod.rs

```diff
@@ -684,6 +697,28 @@ mod tests {
         Ok(())
     }
 
+    #[tokio::test]
+    async fn get_job_status_raw() -> Result<()> {
+        let body = todo!();
```

Comment:
WIP?

Reply:
Forgot to push the latest changes, on second.